frontend/packages/console-shared/src/components/pod/PodRing.tsx

```diff
@@ -54,7 +54,8 @@ const PodRing: React.FC<PodRingProps> = ({
   const handleScaling = _.debounce(
     (operation: number) => {
       const patch = [{ op: 'replace', path, value: operation }];
-      const promise: Promise<K8sResourceKind> = k8sPatch(resourceKind, obj, patch);
+      const opts = { path: 'scale' };
```

Comment:
You'll need to update the pod ring access check as well to use the `scale` subresource.

Reply:
This right?  https://github.com/openshift/console/pull/7807/commits/335306b2a232e861d5b88c83d9c6c9e033e88d59

Reply:
@rhamilto Looks right